logging/src/main.ts

```diff
@@ -1,9 +1,21 @@
 import express, { Express, NextFunction, Request, Response } from "express";
 import { request, ClientRequest, RequestOptions, IncomingMessage } from "http";
+import {writeStatusToDisk} from "./status";
```

Comment:
Can I be annoying and ask you to run Prettier on this file? Formatting is quite different in places